snmp/ups-apcups.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+################################################################
+# copy this script to somewhere like /opt and make chmod +x it #
+# edit your snmpd.conf and include                             #
+# extend ups-apcups /opt/ups-apcups.sh                         #
```

Comment:
The suggestion from docs is for agent setup, maybe we should put this straight now, and I'll change the docs and all other. I suggest something like /opt/librenms/apps or /etc/snmpd/extend